common/psbt_open.c

```diff
@@ -89,8 +91,12 @@ static const u8 *linearize_output(const tal_t *ctx,
 
 	if (wally_tx_add_output(psbt->tx, tx_out) != WALLY_OK)
 		abort();
+
 	psbt->outputs[0] = *out;
 	psbt->num_outputs++;
+	/* Blank out unknowns. These are unordered and serializing
+	 * them might have different outputs for identical data */
+	psbt->outputs[0].unknowns.num_items = 0;
```

Comment:
wally_map_sort(&psbt->outputs[0].unknowns) can be used now that its been merged to wally master.